crates/storage/provider/src/providers/static_file/manager.rs

```diff
@@ -74,6 +74,53 @@ impl StaticFileAccess {
     }
 }
 
+/// Result of file-level consistency checks for static files.
+///
+/// This is an opaque token returned by [`StaticFileProvider::check_file_consistency`]
+/// that should be passed to [`StaticFileProvider::check_storage_consistency`].
+///
+/// Contains the healed state for each segment after file-level recovery,
+/// and any unwind target detected during file healing.
+#[derive(Debug, Default)]
```

Comment:
is the `Default` derive required? let's remove if not, it can be a footgun to allow external consumers to create an empty `FileConsistencyResult` (`segment_state` would be empty and every segment would be skipped, no storage consistency would be run)